internal/services/orbital/spacecraft_resource_test.go

```diff
@@ -0,0 +1,172 @@
+			),
+		},
+		{
+			Config: r.update(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+func TestAccSpacecraft_complete(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_spacecraft", "test")
+	r := SpacecraftResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.complete(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+func (r SpacecraftResource) Exists(ctx context.Context, client *clients.Client, state *pluginsdk.InstanceState) (*bool, error) {
+	id, err := spacecraft.ParseSpacecraftID(state.ID)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := client.Orbital.SpacecraftClient.Get(ctx, *id)
+	if err != nil {
+		if response.WasNotFound(resp.HttpResponse) {
+			return utils.Bool(false), nil
+		}
+		return nil, fmt.Errorf("retrieving %s: %+v", *id, err)
+	}
+	return utils.Bool(true), nil
+}
+
+func (r SpacecraftResource) basic(data acceptance.TestData) string {
+	template := r.template(data)
+	return fmt.Sprintf(`
+%[1]s
+
+resource "azurerm_spacecraft" "test" {
+  name                = "acctestspacecraft-%[2]d"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = "westus"
+  norad_id            = "12345"
+  links {
+    bandwidth_mhz        = 100
+    center_frequency_mhz = 101
+    direction            = "Uplink"
+    polarization         = "LHCP"
+    name                 = "linkname"
+  }
+  tle_line_1 = "1 23455U 94089A   97320.90946019  .00000140  00000-0  10191-3 0  2621"
+  tle_line_2 = "2 23455  99.0090 272.6745 0008546 223.1686 136.8816 14.11711747148495"
+  title_line = "AQUA"
+}
+`, template, data.RandomInteger)
+}
+
+func (r SpacecraftResource) update(data acceptance.TestData) string {
+	template := r.template(data)
+	return fmt.Sprintf(`
+%[1]s
+
+resource "azurerm_spacecraft" "test" {
+  name                = "acctestspacecraft-%[2]d"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = "eastus"
+  norad_id            = "23456"
+  links {
+    bandwidth_mhz        = 100
+    center_frequency_mhz = 101
+    direction            = "Uplink"
+    polarization         = "LHCP"
+    name                 = "linkname"
+  }
+  tle_line_1 = "1 23455U 94089A   97320.90946019  .00000140  00000-0  10191-3 0  2621"
+  tle_line_2 = "2 23455  99.0090 272.6745 0008546 223.1686 136.8816 14.11711747148495"
+  title_line = "AQUB"
+}
+`, template, data.RandomInteger)
+}
+
+func (r SpacecraftResource) complete(data acceptance.TestData) string {
+	template := r.template(data)
+	return fmt.Sprintf(`
+%[1]s
+
+resource "azurerm_spacecraft" "test" {
+  name                = "acctestspacecraft-%[2]d"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = "eastus"
+  norad_id            = "12345"
+  links {
```

Comment:
tests should be formatted to be readable:
```suggestion
  norad_id            = "12345"

  links {
```

Reply:
Will reformat.